web/src/feature/calculation_input/TempOrPressureInput.tsx

```diff
@@ -34,9 +39,14 @@ export const TempOrPressureInput = (props: {
           ? 'The ' + props.name.toLowerCase() + ' is very low'
           : undefined
       }
-      onChange={(event: React.ChangeEvent<HTMLInputElement>) =>
+      onChange={(event: React.ChangeEvent<HTMLInputElement>) => {
+        const regex = /^[-+]?\d+(\.\d+)?([eE][-+]?\d+)?$/
```

Comment:
Could we add a short description to the regex this?
What it will allow, deny, etc.